### DISCUSSION TITLE
5.0版本，引入tooltip插件，低版本浏览器会提示路径不对的报错，怎么解决

### DISCUSSION THREAD

**Discussion by carrotming** [antvis/G6#6085] [General]
5.0版本，引入tooltip插件，低版本浏览器会提示路径不对的报错，怎么解决

**Comment by hustcc** [comment]
具体是什么浏览器版本，报错截图有吗？

**Reply by carrotming** [reply]
![谷歌85版本报错](https://github.com/user-attachments/assets/d6bdd303-c557-4837-bfc0-6e9810f80610)
![谷歌85dev环境报错](https://github.com/user-attachments/assets/70141417-2796-466a-8466-26e531da3089)
![谷歌打包85版本报错](https://github.com/user-attachments/assets/312b03d9-1290-4b22-833d-d70bb5b257b4)
你好。这里有三张图片，前两张是在run dev模式报的错，最后一张是打包报的错
谷歌浏览器版本85，我用107测试过可以。
我用的是vue3,vite4.5版本
如果可以的话，告诉我怎么解决。因为我不想降低版本
antv4.8和antv5.0不兼容。
这里报了两个错，一个是插件tooltip的错，一个是getZoom的错。
我不知道是否需要非常高的浏览器版本

**Comment by Jiaocz** [comment]
可以看下是怎么引入的吗
